site/content/en/docs/Advanced/allocator-service.md

```diff
@@ -35,15 +36,18 @@ If the `agones-allocator` service is installed as a `LoadBalancer` [using a stat
 
 Replace the default server TLS certificate with a certificate with CN and subjectAltName. There are multiple approaches to generate a certificate. Agones recommends using [cert-manager.io](https://cert-manager.io/) solution for cluster level certificate management.
 
-In order to use cert-manager solution, first, [install cert-manager](https://cert-manager.io/docs/installation/kubernetes/) on the cluster. Then, [configure](https://cert-manager.io/docs/configuration/) an `Issuer`/`ClusterIssuer` resource and last configure a `Certificate` resource to manage allocator-tls `Secret`.
+In order to use the cert-manager solution, first [install cert-manager](https://cert-manager.io/docs/installation/kubernetes/) on the cluster.
+Then, [configure](https://cert-manager.io/docs/configuration/) an `Issuer`/`ClusterIssuer` resource and
+last [configure](https://cert-manager.io/docs/usage/certificate/) a `Certificate` resource to manage allocator-tls `Secret`.
+Make sure to configure the `Certificate` based on your system's requirements, including the validity `duration`.
 
 Here is an example of using a self-signed `ClusterIssuer` for configuring allocator-tls `Secret`:
 
 ```bash
 #!/bin/bash
 # Create a self-signed ClusterIssuer
 cat <<EOF | kubectl apply -f -
-apiVersion: cert-manager.io/v1alpha2
+apiVersion: cert-manager.io/v1
```

Comment:
Can you ack that it is ok to publish these changes for the existing release (vs. gating them on the next agones release)? I think that it should be ok, but want to double check.

Reply:
They are ok, unless a cluster is using an old version of cert-manager, which is independent to Agones version.
I manually tested it on Agones v1.6 cluster.